modules/rest-api/src/blaze/rest_api/routes.clj

```diff
@@ -64,6 +64,7 @@
    :compile (fn [{:keys [response-type] :response-type.json/keys [opts]} _]
               (condp = response-type
                 :json (output/wrap-output opts)
+                :binary (fhir-output/wrap-binary-output opts)
```

Comment:
here, you have to return a middleware function (a function taking a handler). It would be possible to write `#(fhir-output/wrap-binary-output % opts)` as this creates a function taking a handler. but as we don't need the opts, we can just return `fhir-output/wrap-binary-output`.